samples/Shell/Fabimals/Fabimals/Views/About.fs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright 2018-2019 Fabulous contributors. See LICENSE.md for license.
+namespace Fabimals.Views
+
+open Fabulous.Core
+open Fabulous.DynamicViews
+open Xamarin.Forms
+open System
+
+module AboutStyles =
+    let primaryColor = Color.FromHex("#96d1ff")
+    let lightTextColor = Color.FromHex("#999999")
+
+module About =
+    type Msg =
+        | ShowFabulous
+        | ShowXamarinForms
+        | ShowOriginalFabimalsSample
+        
+    type CmdMsg = GoToUrl of string
+
+    let goToUrl url =
+        Device.OpenUri(new Uri(url))
+        Cmd.none
+
+    let mapCommands cmdMsg =
+        match cmdMsg with
+        | GoToUrl url -> goToUrl url
+
+    let update msg =
+        match msg with
+        | ShowFabulous -> [GoToUrl "https://fsprojects.github.io/Fabulous"]
+        | ShowXamarinForms -> [GoToUrl "https://docs.microsoft.com/en-us/xamarin/xamarin-forms"]
+        | ShowOriginalFabimalsSample -> [GoToUrl "https://github.com/xamarin/xamarin-forms-samples/blob/master/UserInterface/Xaminals"]
+
+    let view dispatch =
+        dependsOn () (fun _ () ->
+            View.ContentPage(
+                title="About",
+                content=View.ScrollView(
+                    View.Grid(
+                        rowdefs=["auto"; "*"],
+                        children=[
+                            View.StackLayout(
```

Comment:
Don't know it for sure but nested Stacklayouts are always heavy in performance. I think Flex or Grid could be better in it?
Generally I think the About page is kinda interesting. You have a Scrollview with a Grid and a scrollview again but I think there is no need for this.

Scrollview and a Grid should be enough?

Reply:
You're right. I'll take a closer look at this page to see what can be simplified.
I only translated the AboutPage.xaml from Xaminals without asking too much questions. :)

Reply:
Maybe this could be interesting: [https://github.com/jsuarezruiz/xamarin-forms-perf-playground](https://github.com/jsuarezruiz/xamarin-forms-perf-playground)

There are some explanation on do's and dont's